src/zed_cpu_ros.cpp

```diff
@@ -31,9 +31,10 @@ class StereoCamera
 	 * @param[in]  resolution  The resolution
 	 * @param[in]  frame_rate  The frame rate
 	 */
-	StereoCamera(int resolution, double frame_rate): frame_rate_(30.0) {
+	 StereoCamera(int device, int resolution, double frame_rate)
+ : frame_rate_(30.0) {
```

Comment:
can you make this on the same line? Thanks

Reply:
Done in 311cb52 :)